examples/FirebaseStream_ESP8266/FirebaseStream_ESP8266.ino

```diff
@@ -42,22 +42,22 @@ void setup() {
   Serial.println();
   Serial.print("connected: ");
   Serial.println(WiFi.localIP());
-  
-  fbase.stream("/bitcoin");
 }
 
 
-void loop() {
-  if (fbase.error()) {
+void loop() {  
+  static FirebaseEventStream stream = fbase.stream("/bitcoin");
```

Comment:
Ya I am still playing with that but it would have to be either a static like this and use return value optimization to allow this to be constructed or to switch over to returning a unique_ptr<FirebaseCall> which is a little ugly but would work like we want.